calico/felix/fetcd.py

```diff
@@ -132,6 +149,67 @@ def _periodically_resync(self):
             gevent.sleep(sleep_time)
             self.force_resync(reason="periodic resync", async=True)
 
+    @logging_exceptions
+    def _status_reporting(self):
```

Comment:
Nitpick: `_status_reporting` isn't a verb phrase.  `_periodically_report_status` is wordy but matches the method above